spec/system/characters/index_spec.rb

```diff
@@ -187,6 +187,11 @@ def expect_character_icons(characters)
 
     include_examples "characters#index", false
 
+    scenario "Handles bad pages" do
+      visit user_characters_path(user_id: user.id, view: 'list', character_split: 'template', page: "nvOpzp; AND 1=1")
+      expect(page).to have_text("Sample user's Characters")
+    end
```

Comment:
huh why is this one a system spec?

Reply:
From Discord: because Marri didn't know enough about the differences between system and request specs to know she should use a request spec, and she will move it.